broker/lua/src/macro_cache.cc

```diff
@@ -17,12 +17,15 @@
  */
 
 #include "com/centreon/broker/lua/macro_cache.hh"
+#include <absl/container/flat_hash_set.h>
+#include <memory>
```

Comment:
Why are they needed?
Are you sure you need them ? I talk about flat_hash_set and memory.
We already talk about them in the header, so they at least should be declared there...

Reply:
will fix